packages/editor-ui/src/components/NodeSettingsTabs.vue

```diff
@@ -45,11 +40,21 @@ export default mixins(
 				return nodeType.documentationUrl;
 			}
 
-			if (nodeType.documentationUrl || (nodeType.name && nodeType.name.startsWith('n8n-nodes-base'))) {
-				return 'https://docs.n8n.io/nodes/' + (nodeType.documentationUrl || nodeType.name) + '?utm_source=n8n_app&utm_medium=node_settings_modal-credential_link&utm_campaign=' + nodeType.name;
+			const utmTags = '?utm_source=n8n_app&utm_medium=node_settings_modal-credential_link' +
+				'&utm_campaign=' + nodeType.name;
+
+			// Built-in node documentation available via its codex entry
+			const primaryDocUrl = nodeType.codex?.resources?.primaryDocumentation?.[0]?.url;
+			if (primaryDocUrl) {
+				return primaryDocUrl + utmTags;
+			}
```

Comment:
All our built-in nodes (with the exception of the StickyNote, I think) have a primary documentation link defined in their descriptor, so let's use that directly where it applies.